conanfile.py

```diff
@@ -9,7 +9,7 @@
 
 class SISLConan(ConanFile):
     name = "sisl"
-    version = "8.9.2"
+    version = "8.9.3"
```

Comment:
Do we need a minor version change or major change? 
If no other repo will pick up 8.9.* automatically, it should be fine.

Reply:
Ideally major version, but we will have to recompile all the dependencies with new sisl version, it should be fine.